sdk/translation/ai-translation-document-rest/review/ai-translation-document.api.md

```diff
@@ -201,6 +189,11 @@ export type DocumentTranslationClient = Client & {
     path: Routes;
 };
 
+// @public
+export interface DocumentTranslationClientOptions extends ClientOptions {
+    apiVersion?: string;
```

Comment:
Is this a known set of values or just any random string will do?

Reply:
FWIW this field was always there (it's also defined in `ClientOptions` I believe, and if not, it's defined somewhere higher up the chain), so there's no material difference in API surface resulting from this. We just specify it here now due to an emitter change where the emitted code is handling the `apiVersion` parameter instead of a Core policy.

Reply:
If I am understanding it correctly, what @mpodwysocki was asking is the value of apiVersion, which is yes, the allowed values for apiVersion should be a known set of values, however, we didn't generate it as string union, because 
1. we have some user cases where a customer uses an older version SDK but passes a newer apiVersion to some apis, we have no way to know a newer apiVersion when generating SDK from an older version spec. And if would be a breaking if we used to generate it as string, but generate it as a limited string union now.
2. when service team converts from swagger to typespec, they will only convert starting from a specific api version, then authoring typespec only for the future api version, they will not convert the older api version swagger into typespec, the api version enum we get from typespec, it's an incomplete supported api versions list.